dash_bio/Ideogram.py

```diff
@@ -113,12 +113,12 @@ class Ideogram(Component):
 }. homology has the following type: dict containing keys 'chrOne', 'chrTwo'.
 Those keys have the following types:
   - chrOne (optional): . chrOne has the following type: dict containing keys 'organism', 'start', 'stop'.
-Those keys have the following types:
+Those keys have the following types: 
```

Comment:
These whitespace diffs might be attributed to the Dash version used to generate the components, too.